aws/versions.tf

```diff
@@ -1,4 +1,7 @@
 terraform {
+   backend "s3" {  
+     # backend file expected to fill in here, vars not allowed 
```

Comment:
So, there is nothing that a user needs to specify here?

Reply:
> Jake - please update the documentation if awsudo is now required.

Absolutely will - it's just not in this project it's in Jupyterhub Deploy so it's not this PR

Reply:
> So, there is nothing that a user needs to specify here?

The way it works is that you need to fill out the backend.conf template file and specify it in the terraform init command now. Note that if you don't do any of that it will just prompt you for the missing info when you run terraform init

Reply:
> Absolutely will - it's just not in this project it's in Jupyterhub Deploy so it's not this PR

Ah, right.  Thanks!